server/src/main/java/org/elasticsearch/index/engine/Segment.java

```diff
@@ -189,6 +189,13 @@ private static Sort readSegmentSort(StreamInput in) throws IOException {
                 if (missingFirst != null) {
                     fields[i].setMissingValue(missingFirst ? SortedSetSortField.STRING_FIRST : SortedSetSortField.STRING_LAST);
                 }
+            } else if (type == 5) {
```

Comment:
Where can I see what `type = 5` is? A comment would help here, a constant would be even better, I think

Reply:
Yeah.  I was following the old way and it didn't have them. I didn't want to rock the boat too much here, but there isn't any reason not to try and at least make it more readable. I'll add some constants.